packages/data-grid/src/quickFilter/QuickFilter.jsx

```diff
@@ -30,11 +30,28 @@ const propTypes = {
 };
 
 function QuickFilter({ filters, onFilterSelect }) {
+  /** Used for the options array when there is no items to filter by */
+  const emptyFilterOption = [
+    {
+      label: 'No items to filter by',
```

Comment:
Do you think it makes sense to name this as disabledFilterOption as apposed to empty? Also the label string needs to be configurable so pass in a prop

Reply:
I personally prefer the `empty` word as opposed to `disabled` because the filter is in fact empty, not disabled...
Sure, the label string can be configurable, good idea...